packages/rocketchat-theme/client/imports/components/modal/full-modal.css

```diff
@@ -9,6 +9,8 @@
 	height: 100%;
 	justify-content: center;
 
+	overflow-y: auto;
```

Comment:
Can I remove this? I fixed that on #8637

Reply:
yes please